src/app/Mappings/mocks/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { MappingType } from '../types';
+
+export const updateMockStorage = (params: any) => {
```

Comment:
We might want to add a comment here that this is temporary, so we don't invest too much time making it better / spreading it.